pyproject.toml

```diff
@@ -30,9 +30,9 @@ dependencies = [
     "scipy>=1.5.2",
     "rustworkx>=0.13.0",
     "qiskit-aer>=0.12.0",
-    "qiskit>=0.44.1",
+    "qiskit==1.0.0b1",
```

Comment:
I would use `1.0.0rc1` here  if you're testing for compatibility with 1.0.0. The `1.0.0b1`  release was just a very early snapshot for the quantum summit and the final api is very different in `rc1`

Reply:
Thank you, I am seeing the more updated things now :)

I think `qiskit>=0.46<2.0` is the right thing to do to get us through the release?